google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts.
+After=local-fs.target network-online.target network.target
+After=google.service rsyslog.service
```

Comment:
In fact you only need "Before=shutdown.target" (shutdown.target is always reached before reboot.target or halt.target, so no need to specify the latter two).